local/config/finna/searches.ini.sample

```diff
@@ -83,6 +83,9 @@ hide_component_parts = true
 ; Maximum result to display
 result_limit = 100000
 
+; Whether to disable result item expanding in condensed view
```

Comment:
Asetus kuuluisi ennemmin searches.ini:in, jossa muutenkin määritellä hakutulosnäkymän ulkoasua ja käyttäytymistä.

Reply:
Asetus on nyt searches.inissä...